plugins/aver-hid/fu-aver-safeisp-device.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2023 Richard Hughes <richard@hughsie.com>
```

Comment:
me?! :)

Reply:
We will update the author information. 😄